src/main/java/com/amazon/janusgraph/graph/AbstractGraph.java

```diff
@@ -0,0 +1,30 @@
+package com.amazon.janusgraph.graph;
+
+import com.amazon.janusgraph.triple.Triple;
+import org.janusgraph.core.JanusGraph;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.util.Set;
+import java.util.concurrent.*;
+
+
+/**
+ * Created by addisonslabaugh on 6/21/17.
+ */
+public interface AbstractGraph {
```

Comment:
the name of this class does not match the way it is used. it is specific to TravelGraph